docs/index.adoc

```diff
@@ -11,10 +11,12 @@ include::constants.adoc[]
 = tBTC: A Decentralized Redeemable BTC-backed ERC-20 Token
 
 ifndef::release[]
-.DRAFT
+.FINAL
```

Comment:
Should we wait for the final label until governance is documented?

Reply:
I would rather just hand off ownership of this PR and have you write the governance section when ready, tbh, but could go either way. I'm a pumpkin for the week on this one. Rest of the changes are in.

Reply:
I'll take it 😄